cdap-data-fabric/src/main/java/co/cask/cdap/data2/registry/UsageTable.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright © 2015-2019 Cask Data, Inc.
```

Comment:
should be 2019

Reply:
Github doesn't show this as a name change for whatever reason